src/sfdxFlags.ts

```diff
@@ -732,8 +737,10 @@ export function buildSfdxFlags(
       if (!isKeyOf(optionalBuiltinFlags, key)) {
         throw messages.createError('error.UnknownBuiltinFlagType', [key]);
       }
+      // @ts-ignore
```

Comment:
I think this shouldn't have to be ignored (every key on Output should be in `optionalBuiltinFlags`